.github/readme.md

```diff
@@ -23,47 +23,93 @@ If you want to contribute as a developer, please contact us in the [Discord](htt
 
 ## Building
 
-First, install the dependencies for Debian-like distributions.
-   
-``sudo apt install build-essential cmake libunwind-dev libglfw3-dev libvulkan-dev vulkan-validationlayers-dev spirv-tools glslang-tools libspirv-cross-c-shared-dev libxbyak-dev``
+
+### The dependencies for Debian-like distributions.
+```   
+sudo apt install build-essential cmake libunwind-dev libglfw3-dev libvulkan-dev vulkan-validationlayers-dev spirv-tools glslang-tools libspirv-cross-c-shared-dev
+```
+
+### The dependencies for Fedora distributions:
+
+```
+sudo dnf install cmake libunwind-devel glfw-devel vulkan-devel vulkan-validation-layers-devel spirv-tools glslang-devel gcc-c++ gcc spirv-tools-devel xbyak-devel
+```
+
+### The dependencies for Arch distributions:
+
+```
+sudo pacman -S libunwind glfw-x11 vulkan-devel glslang
+```
+
+## Getting spriv-cross on Fedora and Arch Linux
+
+```
+git clone https://github.com/KhronosGroup/SPIRV-Cross && cd SPIRV-Cross && mkdir build && cd build && cmake .. && cmake --build . && sudo make install
+```
+> **Warning** <br/>
+> Fedora will compile to a point and then error out
+> Arch will have to use xbyak from the aur for now
 
 ## Cloning the Repo
 
-``git clone https://github.com/RPCSX/rpcsx && cd rpcsx``
+```
+git clone --recursive https://github.com/RPCSX/rpcsx && cd rpcsx
+```
+
+> if you get a cmake error run
+```
+git submodule update --init --recursive
+```
    
-## Building the emulator
+## How to compile the emulator
    
-`mkdir -p build && cd build && cmake .. && cmake --build .`
+```
+mkdir -p build && cd build && cmake .. && cmake --build .
+```
 
 ## How to create a Virtual HDD
 
 > The PS4 has a case-insensitive filesystem. To create the Virtual HDD, do the following:
  
-`truncate -s 512M ps4-hdd.exfat`
+```
+truncate -s 512M ps4-hdd.exfat
+```
 
-`mkfs.exfat -n PS4-HDD ./ps4-hdd.exfat`
+```
+mkfs.exfat -n PS4-HDD ./ps4-hdd.exfat
+```
 
-`mkdir ps4-fs`
+```
+mkdir ps4-fs
+```
 
-``sudo mount -t exfat -o uid=`id -u`,gid=`id -g` ./ps4-hdd.exfat ./ps4-fs``
+```
+sudo mount -t exfat -o uid=`id -u`,gid=`id -g` ./ps4-hdd.exfat ./ps4-fs
+```
```

Comment:
is it possible to have all of these in a single block or in a one-liner?
it looks very chunky when rendered

Reply:
great idea